### ISSUE TITLE
Loss scale is abnormally low during T2V training with wan

### ISSUE THREAD

**Issue by wileewang** [tdrussell/diffusion-pipe#187]
**Description**  
When training with `wan` for text-to-video (T2V), I noticed that the loss scale remains around `0.01` or even lower, which seems abnormally small.

However, when using a different fine-tuning codebase for `wan`, the loss scale typically falls within the expected range of `0.1 - 0.7`. This raises concerns that there may be an issue with the current training configuration or implementation.

**Questions**  
- What could be causing the loss scale to remain so low?
- Could this be related to AMP (Automatic Mixed Precision), gradient accumulation, optimizer settings, or other factors?
- Will such a low loss scale negatively affect model convergence or stability?

**Reference**
```bash
[2025-04-02 13:50:16,938] [INFO] [logging.py:128:log_dist] [Rank 0] step=1, skipped=0, lr=[9.950000000000002e-07], mom=[0.0]
steps: 1 loss: 0.0054 iter time (s): 3.180 samples/sec: 0.314

[2025-04-02 13:50:19,549] [INFO] [logging.py:128:log_dist] [Rank 0] step=2, skipped=0, lr=[1.4900000000000003e-06], mom=[0.0]
steps: 2 loss: 0.0084 iter time (s): 2.553 samples/sec: 0.392

[2025-04-02 13:50:21,963] [INFO] [logging.py:128:log_dist] [Rank 0] step=3, skipped=0, lr=[1.9850000000000005e-06], mom=[0.0]
steps: 3 loss: 0.0051 iter time (s): 2.362 samples/sec: 0.423

[2025-04-02 13:50:24,377] [INFO] [logging.py:128:log_dist] [Rank 0] step=4, skipped=0, lr=[2.480000000000001e-06], mom=[0.0]
steps: 4 loss: 0.0054 iter time (s): 2.360 samples/sec: 0.424

[2025-04-02 13:50:26,787] [INFO] [logging.py:128:log_dist] [Rank 0] step=5, skipped=0, lr=[2.975000000000001e-06], mom=[0.0]
steps: 5 loss: 0.0124 iter time (s): 2.360 samples/sec: 0.424

[2025-04-02 13:50:29,203] [INFO] [logging.py:128:log_dist] [Rank 0] step=6, skipped=0, lr=[3.4700000000000015e-06], mom=[0.0]
steps: 6 loss: 0.0051 iter time (s): 2.363 samples/sec: 0.423

[2025-04-02 13:50:31,615] [INFO] [logging.py:128:log_dist] [Rank 0] step=7, skipped=0, lr=[3.965000000000002e-06], mom=[0.0]
steps: 7 loss: 0.0048 iter time (s): 2.360 samples/sec: 0.424

[2025-04-02 13:50:34,031] [INFO] [logging.py:128:log_dist] [Rank 0] step=8, skipped=0, lr=[4.460000000000002e-06], mom=[0.0]
steps: 8 loss: 0.0277 iter time (s): 2.363 samples/sec: 0.423

[2025-04-02 13:50:36,446] [INFO] [logging.py:128:log_dist] [Rank 0] step=9, skipped=0, lr=[4.9550000000000025e-06], mom=[0.0]
steps: 9 loss: 0.0049 iter time (s): 2.363 samples/sec: 0.423
```

**Comment by tdrussell** [comment]
This does seem wrong. For both image training and video training of the 14b t2v model, I am seeing average loss values in the 0.10 to 0.12 range.

Things to check:
1. What resolution are you training at? Higher res means lower loss because the signal-to-noise is innately higher when the res is higher.
2. Is there anything odd or atypical about your dataset?
3. Make sure you aren't accidentally training the i2v. i2v loss is much lower because the model has access to the first frame to help it make better predictions.
4. Are you doing anything weird like setting the flow shift to an extreme value? (if you have no idea what this is, you aren't doing this)

**Comment by wileewang** [comment]
Thanks for the suggestions!

1. I'm training at a resolution of **832 x 480**, so it's relatively high and shouldn’t be causing unusually high loss.

2. One notable aspect of my dataset is that **all the videos are black and white**, which might be contributing to the loss difference compared to typical RGB datasets.

3. I’ve double-checked, and I’m definitely training the **T2V model**, not I2V.

4. I also confirmed that there’s **nothing unusual with the flow shift setting**—it’s at the default value.

**Comment by tdrussell** [comment]
It might be normal then, and caused by the combination of max resolution and black and white videos. Depending on what your videos are of, it may also just be innately easier for the model to predict.

When you say a different codebase has larger loss values, do you mean you tried your specific dataset, with all of the same settings, and the loss was higher? If so, what is the training code you used? Are you able to share a config for them? I ask because I know, for example, musubi tuner recommends a couple of settings differently in the example commands.

**Comment by wileewang** [comment]
Thanks for the help! The issue was indeed data-related — after trying some more typical data, the loss looks more reasonable.
